tutorials/preprocessing/plot_35_artifact_correction_regression.py

```diff
@@ -5,20 +5,37 @@
 Repairing artifacts with regression
 ===================================
 
-This tutorial covers removal of artifacts using regression as in
-:footcite:`GrattonEtAl1983`.
+This tutorial covers removal of artifacts using regression as in Gratton et al.
+1983 :footcite:`GrattonEtAl1983`.
 
 .. contents:: Page contents
    :local:
    :depth: 2
 
+Generally speaking, artifacts that result in time waveforms on the sensors
+that are accurately reflected by some reference signal can be removed by
+regression. One example of this is blink artifacts captured by (typically
+bipolar) EOG channels, which we will examine here.
+
+Although ECG signals are typically well captured by ECG electrodes,
+regression-based removal usually does not perform very well.
+This is likely because the heart acts like a rotating dipole, and
+therefore the ECG channel time waveform (typically based on a bipolar
+ECG recording) does not reflect the same temporal dynamics that manifest at
+each MEG channel (obtained by sampling some component of the related magnetic
+vector field). Other approaches like :ref:`ICA <tut-artifact-ica>` or
+:ref:`SSP <tut-artifact-ssp>` will likely work better for ECG.
+
```

Comment:
The introduction contains the word "typically" three times - can you use synonyms?